### PR TITLE
Bump images for k/k master to go1.10.4

### PR DESCRIPTION
**Pull request by amwat**
xref: https://github.com/kubernetes/kubernetes/pull/68802

/cc @BenTheElder @ixdy 
/area images
/hold

### CODE REVIEW

File: images/kubekins-e2e/Makefile [hunk 1]
```diff
@@ -17,23 +17,23 @@ TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
...
+GO ?= 1.10.4
 BAZEL ?= 0.16.1
 UPGRADE_DOCKER ?=false
 # TODO(bentheelder,cblecker): this is aging and bad
 CFSSL ?= R1.2
 
 # config for testing prior to rolling out to master / ongoing release
 ifeq ($(K8S), experimental)
-	GO = 1.10.2
+	GO = 1.10.4
```
Comment (BenTheElder) [inline images/kubekins-e2e/Makefile]: as mentioned, we should upgrade this to the latest stable go, but i'll do this in another PR, experimental images are orthogonal here.

**Comment by krzyzacy** [comment]
/hold
bump bootstrap?

**Comment by krzyzacy** [comment]
(or guess it's *fine* for OSS)

**Comment by tpepper** [comment]
I'm a bit worried by the size of storage merge for v1.12 today.  I want to see the test results there still showing ok this morning, and then intend to unhold this PR this afternoon.  Need the team to watch test results especially closely for a good 72hrs after this as it may have not just simpler immediate master branch and quickly following release-1.12 branch issues, but long running scalability issues.

/cc @wojtek-t @shyamjvs

**Comment by BenTheElder** [comment]
This will only go in if we go to put in https://github.com/kubernetes/kubernetes/pull/68802 which is not currently in the milestone. I think probably we do want to put both in though.

**Comment by cblecker** [comment]
FYI, https://github.com/kubernetes/kubernetes/pull/68802 was just unblocked for merge into k/k. Will be merged after retests. Please stand by :)

**Comment by BenTheElder** [comment]
:eyes:

**Pull request merged**